### ISSUE TITLE
docker composition should load all VFs during install

### ISSUE THREAD

**Issue by kevinkle** [superphy/backend#227]
Namely, using VFs from https://github.com/phac-nml/ecoli_vf/blob/master/data/repaired_ecoli_vfs_shortnames.ffn

This is related to https://github.com/superphy/backend/issues/202, to ensure that all Stx1, Stx2 and Eae markers are in the DB. e.g. so someone doing a fresh install can still use phylotyper w/o running any of the ECTyper tasks.

**Comment by kevinkle** [comment]
Needs to be addressed to complete https://github.com/superphy/backend/issues/202

**Comment by kevinkle** [comment]
I'm going to embed this into `superphy_subtyping.ttl` in phylotyper (on backend)

**Comment by kevinkle** [comment]
This should also resolve phylotyper's dependence on submitting an ECTyper task before running

**Comment by kevinkle** [comment]
https://github.com/superphy/backend/commit/6b6e3ad97e99e7297aff8f309d1083fe9817d94e shows how we addressed this. When phylotyper does its check for ontology definitions in the db., it will also upload a graph of all the VFs if the definition isnt found.

This is different from the title of this issue: the problem is resolved within the phylotyper module instead of the docker composition.

Closing issue. https://github.com/superphy/backend/issues/228 will cover the removal of the ectyper job enqueu from phylotyper
